main.go

```diff
@@ -118,11 +118,19 @@ func main() {
 		appconfig.Instance.ScheduleClosing(logger)
 	}
 
+	syncServiceType := viper.GetString("synchronization_service.type")
+	syncServiceEndpoint := viper.GetString("synchronization_service.endpoint")
+	viper.SetDefault("synchronization_service.connection_timeout_seconds", 20)
```

Comment:
It would be better to set default values in one place (appconfig.setDefaultParams())

Reply:
Thanks, fixed